Add `Beatmap/Pattern/osu!mania` listing and subpages (#8595)
ppy/osu-wiki: wiki/Beatmap/Pattern/Mania/Long Notes/en.md

---
stub: true
tags:
  - jacks
---

# Long Notes

> Also referred to in-game as holds, **long notes** are notes that require you to hold and release them afterwards. They are often referred to as LNs for short and are used to represent prolonged sounds in the music.

Reviewer: Maybe add that some people call them noodles, similar with rice on notes.

Author: ```suggestion
Also referred to as holds in the editor, **long notes** are notes that require you to hold and release them afterwards. They are often referred to as noodles or LNs and are used to represent prolonged sounds in the music.
```

![](img/ln.png)

> **Releases** are known to be their own term. It refers to the ending of a **long note**, the part that requires you to release the key.

Reviewer: I would maybe add:

`This term is frequently used because osu! takes release timing in consideration when awarding a judgement for the full long note`

Or something that is redacted better.

Author: ```suggestion
**Releases** refers to the ending of a **long note**, the part that requires you to release the key. They are known to have their own term due to release timing in osu!mania being taken into consideration when awarding judgements for the entire long note.
```

## Shields

**Shields** occur when a note is placed consecutively before a long note in the same column. They play very similarly to [minijacks](/wiki/Beatmap/Pattern/Mania/Jacks/en.md#Minijack).

![](img/shield.png)

> **Reverse shields** occur when a consecutive note follows a long note's release.

Reviewer: One of them clarify that they are in the same column but the other one does not. I would say mentioning the same column for both is better but being consistent should be enough.

Author: ```suggestion
**Shields** occur when a note is placed consecutively before a long note in the same column. They play very similarly to [minijacks](/wiki/Beatmap/Pattern/Mania/Jacks/en.md#Minijack).

![](img/shield.png)

**Reverse shields** occur when a consecutive note follows a long note's release within the same column.
```
